<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>900</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(210, 208, 208);</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>580</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 16pt &quot;Segoe UI Semibold&quot;;
color:white;
background-color: rgb(148, 176, 182);
</string>
    </property>
    <property name="text">
     <string>WYKRYTE DEFEKTY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="analyze_button">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>730</y>
      <width>211</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>rozpocznij analizę</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>730</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QLabel" name="cap">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(183, 202, 255);</string>
    </property>
    <property name="text">
     <string>Kamera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="virtual_cam_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>730</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Włącz wirtualną kamerę</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_button">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>730</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="record_button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>730</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>nagraj ekran</string>
    </property>
   </widget>
   <widget class="QComboBox" name="component">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>780</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>780</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Wybierz komponent</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>840</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>liczba wykrytych elementów: </string>
    </property>
   </widget>
   <widget class="QLabel" name="count_elements">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>840</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="component_list">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>740</y>
      <width>256</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 127);</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>cap</zorder>
   <zorder>analyze_button</zorder>
   <zorder>start_button</zorder>
   <zorder>virtual_cam_button</zorder>
   <zorder>stop_button</zorder>
   <zorder>record_button</zorder>
   <zorder>component</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>count_elements</zorder>
   <zorder>component_list</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
